redhat-cop/infra-ansible
# Add initial aws security group management

### What does this PR do?
Allows for creation/management of AWS security groups

### How should this be tested?
Update sample inventory to use valid aws credentials and the security group of your choice. Then run the playbook `secgroups.yml` underneath tests. You should then be able to run it / rerun it to show that it creates the network.

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
#323 

### People to notify
cc: @redhat-cop/infra-ansible
